corehq/apps/domain/templates/domain/admin/flags_and_privileges.html

```diff
@@ -87,45 +95,48 @@ <h1>Feature Flags</h1>
             </p>
         </div>
         <div class="col-sm-10">
-            <table class="table table-striped">
+            <table class="table table-striped" id="toggles-table">
                 <thead>
                     <th>{% trans "Tag" %}</th>
                     <th>{% trans "Feature" %}</th>
                     <th class="checkmark-column">{% trans "Enabled for domain?" %}</th>
                     <th class="checkmark-column">{% trans "Enabled for me?" %}</th>
                     <th>{% trans "Edit" %}</th>
                 </thead>
-                <tbody>
-                    {% for toggle, enabled_for_domain, enabled_for_user in flags %}
-                    <tr class="{% if enabled_for_domain %}success{% elif enabled_for_user %}info{% endif %}">
-                        <td><span class="label label-{{ toggle.tag.css_class }}">{{ toggle.tag.name }}</span></td>
-                        <td>
+                <tbody data-bind="foreach: toggles">
+                    <tr data-bind="css: cssClass">
+                        <td><span class="label"
+                                  data-bind="css: ('label-' + tagCssClass),
+                                             text: tag"></span></td>
 
-                            {% if toggle.description %}
+                        <td>
                             <a class="collapsed" data-toggle="collapse" aria-expanded="false"
-                               href="#toggleDescription-{{ toggle.slug }}">
+                               data-bind="visible: description,
+                                          attr: {href: '#toggleDescription-' + slug}">
                                 <i class="fa fa-angle-double-down"></i>
                             </a>
-                            {% endif %}
 
-                            {{ toggle.label }}
+                            <span data-bind="text: label"></span>
 
-                            {% if toggle.help_link %} (<a href="{{ toggle.help_link }}" target="_blank">docs</a>){% endif %}
+                            <span data-bind="visible: helpLink">
+                                (<a data-bind="attr: {href: helpLink}" target="_blank">docs</a>)
+                            </span>
 
-                            {% if toggle.description %}
-                            <div class="collapse" id="toggleDescription-{{ toggle.slug }}">
-                                <div class="well">
-                                    {{ toggle.description }}
-                                </div>
+                            <div class="collapse"
+                                 data-bind="visible: description,
+                                            attr: {id: 'toggleDescription-' + slug}">
+                                <div class="well" data-bind="text: description"></div>
                             </div>
-                            {% endif %}
+                        </td>
 
+                        <td class="checkmark-column">
```

Comment:
You didn't introduce this, but the HQ-specific `checkmark-class` could be replaced with bootstrap's `text-center` and the style block at the top of this file could be deleted.

Reply:
:+1: